sympy/functions/special/spherical_harmonics.py

```diff
@@ -103,8 +104,8 @@ class Ynm(Function):
 
     >>> from sympy import Ynm, Symbol, conjugate
     >>> from sympy.abc import n,m
-    >>> theta = Symbol("theta")
-    >>> phi = Symbol("phi")
+    >>> theta = Symbol("theta", real=True)
+    >>> phi = Symbol("phi", real=True)
```

Comment:
Similar here. `Ynm(n, m, theta, phi).conjugate()` should work. (And no additional imports or recomputations.)

I may be wrong though, but I am under the impression that intermediate text does not require new imports and redefinitions...